guacamole/src/main/webapp/app/form/templates/dateField.html

```diff
@@ -8,5 +8,6 @@
            guac-focus="focused"
            placeholder="{{'FORM.FIELD_PLACEHOLDER_DATE' | translate}}"
            autocorrect="off"
-           autocapitalize="off"/>
+           autocapitalize="off"
+           autocomplete="off"/>
```

Comment:
Any idea whether these input fields will also need proper `name` attributes for browser autocomplete to function reliably?

The MDN article on disabling autocomplete across the board has a section at the bottom titled ["Tools for disabling autocompletion"](Tools_for_disabling_autocompletion) which seems to imply that the `name` attribute plays an important role:

> ## Tools for disabling autocompletion
>
> The [`jquery.disableAutoFill`](https://terrylinooo.github.io/jquery.disableAutoFill/) plugin randomizes an input's `name` attribute by default. When the form is submitted, the plugin restores the original name. This prevents auto-completion for all browsers (includes third-party auto-completion extensions) but doesn't necessarily help with login fields.